components/usage/pkg/apiv1/usage.go

```diff
@@ -195,6 +195,7 @@ func dbCostCenterToAPI(c db.CostCenter) *v1.CostCenter {
 		SpendingLimit:   c.SpendingLimit,
 		BillingStrategy: convertBillingStrategyToAPI(c.BillingStrategy),
 		NextBillingTime: timestamppb.New(c.NextBillingTime.Time()),
+		CreationTime:    timestamppb.New(c.CreationTime.Time()),
```

Comment:
The "CreationTime" doesn't denote the current billingCycle start as it gets reset on every cost center update, for instance when I change the spending limit. I think we need to introduce an explicit BillingCycleStart time.

Reply:
Ah, good point -- e.g. I guess that with the current PR, the start date gets reset every time you update the usage limit.